src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/behavior/infer_request/cancellation.cpp

```diff
@@ -8,11 +8,10 @@
 namespace {
 using namespace BehaviorTestsDefinitions;
 using namespace ConformanceTests;
-using namespace ov::test::conformance;
 
 INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, InferRequestCancellationTests,
                          ::testing::Combine(
                                  ::testing::Values(targetDevice),
-                                 ::testing::ValuesIn(emptyConfig)),
+                                 ::testing::ValuesIn(std::vector<std::map<std::string, std::string>>{})),
```

Comment:
why don't we name short-cut for empty config?

Reply:
It was also used in test with old API